paperspace/constants.py

```diff
@@ -0,0 +1,3 @@
+EXPERIMENT_TYPE_SINGLE_NODE_ID = 1
+EXPERIMENT_TYPE_GRPC_MULTI_NODE_ID = 2
```

Comment:
check comment above

Reply:
This is literally copy-pasted from ps-experiments